server/db/models/Gig.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (sequelize, DataTypes) {
+  var Gig = sequelize.define('Gig', {
+    title: DataTypes.STRING,
+    type: DataTypes.ENUM('wedding', 'birthday', 'business'),
```

Comment:
Yea good call. I can normalize this into a separate table and submit a follow-up. That'll end up being the correct long term implementation.